src/routes/events/components/EventList.js

```diff
@@ -1,87 +1,48 @@
-    isFirst: PropTypes.bool.isRequired
-  }
-
-  getUrl () {
-    return `${this.props.parentUrl}/${this.props.event.id}`
-  }
-
-  getEventPlaceholder () {
-    return (
-      this.props.thumbnailPlaceholder === 0 ? EventPlaceholder1
-        : this.props.thumbnailPlaceholder === 1 ? EventPlaceholder2
-          : EventPlaceholder3
-    )
-  }
-
-  render () {
-    const dateModel = this.props.event.dateModel
-    return (
-      <Link href={this.getUrl()}>
-        <div className={this.props.isFirst ? cx(style.firstEvent, style.event) : style.event}>
-          <img className={style.eventThumbnail} src={this.props.event.thumbnail || this.getEventPlaceholder()}/>
-          <div className={style.eventDescription}>
-            <div className={style.eventTitle}>{this.props.event.title}</div>
-            <div className={style.eventDate}>
-              <Timespan startDate={dateModel.startDate}
-                        endDate={dateModel.endDate}
-                        locale={this.props.language}/>
-              , {this.props.event.address}</div>
-            <RemoteContent dangerouslySetInnerHTML={{__html: this.props.event.excerpt.slice(0, 70) + '...'}}/>
-          </div>
-        </div>
-      </Link>
-    )
-  }
-}
-
+/**
+ * Display a list of events
+ */
 class EventList extends React.Component {
   static propTypes = {
-    events: PropTypes.arrayOf(PropTypes.shape({
-      event: PropTypes.instanceOf(EventModel).isRequired,
-      thumbnailPlaceholder: PropTypes.number.isRequired
-    })).isRequired,
+    events: PropTypes.arrayOf(PropTypes.instanceOf(EventModel)).isRequired,
     url: PropTypes.string.isRequired,
     language: PropTypes.string.isRequired
   }
 
   render () {
     const {t} = this.props
+
+    if (!this.props.events || this.props.events.length === 0) {
+      return (
+        <div>
+          <Caption title={t('news')} />
+          <div className={style.noEvents}>
+            {t('currentlyNoEvents')}
+          </div>
+        </div>
+      )
+    }
+
+    const elements = this.props.events.map((event, index) =>
+      <EventListElement key={event.id}
+                        event={event}
+                        parentUrl={this.props.url}
+                        language={this.props.language}
+                        isFirst={index === 0} />
+    )
     return (
       <div className={style.list}>
-        <Caption title={t('news')}/>
-        {this.props.events && this.props.events.length !== 0
-          ? this.props.events.map((event, index) =>
-            <Event key={event.event.id}
-                   event={event.event}
-                   parentUrl={this.props.url}
-                   thumbnailPlaceholder={event.thumbnailPlaceholder}
-                   language={this.props.language}
-                   isFirst={index === 0}/>)
-          : <div className={style.noEvents}>{t('currentlyNoEvents')}</div>
-        }
+        <Caption title={t('news')} />
+        {elements}
       </div>
```

Comment:
Maybe you can replace this by 
```javascript
return (
  <React.Fragment>
     <Caption title={t('news')] />
     <div className={style.list}>
       {elements}
     </div>
  </React.Fragment>
)
```
then you can replace `.first-event { border-top: 2px solid var(--theme-color); }` by `.list { border-bottom: 2px solid var(--theme-color); }` in the stylesheet. That would make more sense. Then you can also remove the `isFirst` logic from the `<Event />`
These are things we would now need to test, so simplifying is in your interest :)